subprojects/gradle/impact-shared/src/main/java/com/avito/impact/configuration/ImplementationConfiguration.kt

```diff
@@ -58,6 +59,10 @@ class ImplementationConfiguration(module: InternalModule) : SimpleConfiguration(
                     .dependencies
                     .withType(DefaultProjectDependency::class.java)
             }
+            .filter {
+                val category = it.attributes.getAttribute(Category.CATEGORY_ATTRIBUTE)?.name
```

Comment:
It is not clear line. Can you explain what is a `Category` and `CATEGORY_ATTRIBUTE`?

Reply:
https://docs.gradle.org/current/javadoc/org/gradle/api/attributes/Category.html